tests/functional/olp-cpp-sdk-dataservice-read/DataserviceReadVersionedLayerClientTest.cpp

```diff
@@ -263,6 +263,98 @@ TEST_F(DataserviceReadVersionedLayerClientTest, Prefetch) {
   }
 }
 
+TEST_F(DataserviceReadVersionedLayerClientTest, PrefetchWrongLevels) {
+  const auto catalog =
+      olp::client::HRN::FromString(CustomParameters::getArgument(
+          "dataservice_read_test_versioned_prefetch_catalog"));
+  const auto kLayerId = CustomParameters::getArgument(
+      "dataservice_read_test_versioned_prefetch_layer");
+  const auto kTileId = CustomParameters::getArgument(
+      "dataservice_read_test_versioned_prefetch_tile");
+
+  auto client = std::make_unique<olp::dataservice::read::VersionedLayerClient>(
+      catalog, kLayerId, *settings_);
+
+  {
+    SCOPED_TRACE("Prefetch tiles online and store them in memory cache");
+    std::vector<olp::geo::TileKey> tile_keys = {
+        olp::geo::TileKey::FromHereTile(kTileId)};
+
+    SCOPED_TRACE("min/max levels are 0");
```

Comment:
The SCOPED_TRACE needs to be under the scope else you will get it when some of the tests below fail also.

```cpp
{
    SCOPED_TRACE("123");
}
{
    SCOPED_TRACE("456");
}
```